cni/network/network.go

```diff
@@ -1258,18 +1259,25 @@ func convertNnsToCniResult(
 
 			intIndex := i
 			for _, ip := range ni.Ipaddresses {
+				ipAddr := net.ParseIP(ip.Ip)
 
-				ipWithPrefix := fmt.Sprintf("%s/%s", ip.Ip, ip.PrefixLength)
-				_, ipNet, err := net.ParseCIDR(ipWithPrefix)
+				var address net.IPNet
+				prefixLength, err := strconv.Atoi(ip.PrefixLength)
```

Comment:
can't we make ip.PrefixLength as integer?

Reply:
ip.PrefixLength is part of nnscontracts so it will be difficult to change it to integer.